src/dp_vnf.c

```diff
@@ -89,6 +149,12 @@ int dp_del_vnf(const uint8_t ul_addr6[DP_VNF_IPV6_ADDR_SIZE])
 		return ret;
 	}
 
+	ret = dp_delete_vnf_value(vnf);
+	if (!DP_FAILED(ret)) {
+		DPGRPC_LOG_WARNING("Cannot delete VNF value record");
```

Comment:
Logging twice can sometimes be beneficial, so this is not a change request, just that you could maybe just log here (where there is more information) and simplify the wrapper for delete/record.